io/FV3GFS_io.F90

```diff
@@ -94,6 +94,8 @@ module FV3GFS_io_mod
   real, parameter:: stndrd_atmos_ps = 101325.0_r8
   real, parameter:: stndrd_atmos_lapse = 0.0065_r8
   real, parameter:: drythresh = 1.e-4_r8, zero = 0.0_r8, one = 1.0_r8
+  real, parameter:: min_lake_orog = 200.0_r8
```

Comment:
Is this the min lake depth? It seems much larger than the actual lake depth in places with 2m or 10m in oro_data. Is it possible to reduce it to, say, 10m?

Reply:
No, it is not lake depth.  It is the minimum orography, i.e. the height of lake from sea-level used to identify lakes.  In most cases this part of the logic is unused.